frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -162,6 +163,13 @@ export const notebookLogic = kea<notebookLogicType>([
                 setEditable: (_, { editable }) => editable,
             },
         ],
+        isShowingSidebar: [
+            false,
+            {
+                setSelectedNodeId: () => false,
```

Comment:
I don't think this should be here anymore. If we are changing to a user initiated opening then I think we should leave it open until either the setting is closed or the source node is deleted.

Reply:
Good call. I made a change to leave the sidebar open except in the case when you click a non-custom node (e.g. paragraph)